pkg/util/locode/db/airports/calls.go

```diff
@@ -0,0 +1,186 @@
+package airportsdb
+
+import (
+	"encoding/csv"
+	"io"
+	"os"
+	"strconv"
+
+	"github.com/nspcc-dev/neofs-node/pkg/util/locode"
+	locodedb "github.com/nspcc-dev/neofs-node/pkg/util/locode/db"
+	"github.com/pkg/errors"
+)
+
+const (
+	_ = iota - 1
+
+	_ // Airport ID
+	_ // Name
+	airportCity
+	airportCountry
+	airportIATA
+	_ // ICAO
+	airportLatitude
+	airportLongitude
+	_ // Altitude
+	_ // Timezone
+	_ // DST
+	_ // Tz database time zone
+	_ // Type
+	_ // Source
```

Comment:
This field order defined in airport database?

Reply:
Yes, it is described [here](https://openflights.org/data.html)